pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -348,27 +349,29 @@ def is_optimized_sync(repository, remote, url):
     return is_optimized
 
 
-def synchronize(remote_pk, repository_pk, mirror, skip_types, optimize, url=None):
+def synchronize(remote_pk, repository_pk, sync_policy, skip_types, optimize, url=None):
```

Comment:
@dralley the tests seem to fail because you removed `mirror` https://github.com/pulp/pulp_rpm/pull/2146/files#diff-37aef788d3a12e65873c374ff05bc8629a8b636cab51ac31c127f94d43f1314eR169 
you also need to update ACS viewsets because the `refresh` actions calls into sync task dispatch